src/Raven.Server/Dashboard/DatabasesInfoNotificationSender.cs

```diff
@@ -274,31 +274,34 @@ private static void DiskUsageCheck(DatabaseInfoCache item, DocumentDatabase data
             DrivesUsage existingDrivesUsage,
             bool disabled)
         {
-            var databaseRecord = serverStore.Cluster.ReadRawDatabase(context, databaseName, out var _);
-            if (databaseRecord == null)
+            using (var rawRecord = serverStore.Cluster.ReadRawDatabaseRecord(context, databaseName))
             {
-                // database doesn't exist
-                return;
-            }
+                if (rawRecord.IsNull())
```

Comment:
`ReadRawDatabaseRecord` should just return `null`. Disposing `null` is a no-op and will not throw.

Reply:
done